/*
 * Use Protocol Buffer version 3.
 */ 
syntax = "proto3";

enum DeliveryStatus
{
    UNKNOWN = 0;
    MSG_OK  = 1;
    MSG_NOK = 2;
}

message DeliveryResult
{
    string userId                         = 1;
    map<string, DeliveryStatus> msgStatus = 2;
}
